kong/tools/utils.lua

```diff
@@ -32,19 +30,12 @@ local split         = pl_stringx.split
 local re_match      = ngx.re.match
 local setmetatable  = setmetatable
 
-ffi.cdef[[
-typedef long time_t;
-typedef int clockid_t;
-typedef struct timespec {
-        time_t   tv_sec;        /* seconds */
-        long     tv_nsec;       /* nanoseconds */
-} nanotime;
-
-int clock_gettime(clockid_t clk_id, struct timespec *tp);
 
+ffi.cdef[[
 int gethostname(char *name, size_t len);
```

Comment:
Since `gethostname` does not be used, could we remove this ffi.cdef block?

Reply:
no, it cannot be removed. It is currently being used in some other places.

Reply:
I found it, in `pdk/node.lua`, we can remove it in another PR.

Reply:
When this PR is merged, I will do it.